opentelemetry-sdk/src/opentelemetry/sdk/_metrics/measurement.py

```diff
@@ -12,6 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from dataclasses import dataclass
+from typing import Optional, Union
 
+from opentelemetry.util.types import Attributes
+
+
+@dataclass(frozen=True)
 class Measurement:
-    pass
+    value: Union[int, float]
+    attributes: Optional[Attributes] = None
```

Comment:
Should also have context

Reply:
`Attributes` is already possibly `None` so not needed.

```suggestion
    attributes: Attributes = None
```